src/LumicalSD.cc

```diff
@@ -57,51 +57,43 @@ void LumicalSD::Initialize(G4HCofThisEvent* hce)
 // --- core of the SD, running whenever a step is inside the assigned volume(s) ------
 G4bool LumicalSD::ProcessHits(G4Step* step, G4TouchableHistory*)
 {
-    //const G4StepPoint* endPoint = step->GetPostStepPoint(); 
+    //const G4StepPoint* endPoint = step->GetPostStepPoint();
     // check that an real interaction occured (eg. not a transportation)
     //G4StepStatus stepStatus = endPoint->GetStepStatus();
     //G4bool transmit = (stepStatus==fGeomBoundary || stepStatus==fWorldBoundary);
     //if (transmit) return true;
 
     //-----------create a hit and populate it with the information ----------
-        
-    G4double edep = (step->GetTotalEnergyDeposit()/MeV) - (step->GetNonIonizingEnergyDeposit()/MeV);
+
+    // G4double edep = (step->GetTotalEnergyDeposit()/MeV) - (step->GetNonIonizingEnergyDeposit()/MeV);
+    G4double edep = (step->GetTotalEnergyDeposit()/MeV);
```

Comment:
I am still not sure that taking out NonIonizingEnergyDeposit is physically correct... I commented it here and this is how I used it in the previous simulation. But this doesn't effect anything, so its just very minor point for later discussions. You can change it back if you feel like it.